packages/orama/tests/tokenizer.test.ts

```diff
@@ -68,7 +68,7 @@ t.test('Tokenizer', async t => {
     const O1 = tokenizer.tokenize(I1, 'english')
     const O2 = tokenizer.tokenize(I2, 'english')
 
-    t.strictSame(O1, ['thi', 'is', 'a', 'test', 'with', 'test', 'duplic'])
+    t.strictSame(O1, ['this', 'is', 'a', 'test', 'with', 'test', 'duplic'])
```

Comment:
I am not sure here. Please add a commit to this PR when necessary.

Reply:
Hi! Yes, this is meant to be truncated due to stemming. Can you please revert this? Everything else looks good to me. Thanks!

Reply:
I see! The computer thinks "this" is the plural of "thi" 🙃

Reply:
Done 🍏